diff --git a/Embedded/DBEngine.cpp b/Embedded/DBEngine.cpp
index f20a1e9ec2..3d5e27a3c4 100644
--- a/Embedded/DBEngine.cpp
+++ b/Embedded/DBEngine.cpp
@@ -148,7 +148,7 @@ class DBEngineImpl : public DBEngine {
 
     try {
       db_handler_ =
-          mapd::make_shared<DBHandler>(prog_config_opts.db_leaves,
+          std::make_shared<DBHandler>(prog_config_opts.db_leaves,
                                        prog_config_opts.string_leaves,
                                        prog_config_opts.base_path,
                                        prog_config_opts.allow_multifrag,
@@ -426,7 +426,7 @@ class DBEngineImpl : public DBEngine {
  private:
   std::string base_path_;
   std::string session_id_;
-  mapd::shared_ptr<DBHandler> db_handler_;
+  std::shared_ptr<DBHandler> db_handler_;
   bool is_temp_db_;
   std::string udf_filename_;
 
